content/accessibility/developer-toolkit.mdx

```diff
@@ -0,0 +1,61 @@
+---
+title: Developer toolkit
+---
+
+A great way to find answers to specific accessibility questions is to do an online search for the component/issue in question + 'a11y', 'accessible', or 'accessibility', and scan for official websites like W3C ([https://w3.org](https://w3.org)). 
```

Comment:
I'm coming back to this intro, do we want to tell developers to Google it at the top of the page? Or should this be a note at the end?
If so, we should have another intro here.